tests/countries/test_guinea.py

```diff
@@ -0,0 +1,232 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.guinea import Guinea, GN, GIN
+from tests.common import CommonCountryTests
+
+
+class TestGuinea(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        years = range(1959, 2050)
+        super().setUpClass(Guinea, years=years)
+        cls.no_estimated_holidays = Guinea(years=years, islamic_show_estimated=False)
+
+    def test_country_aliases(self):
+        self.assertAliases(Guinea, GN, GIN)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Guinea(years=1958))
+
+    def test_new_years_day(self):
+        name = "Fête du Nouvel an"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(1959, 2050)))
+        dt = (
+            "2034-01-02",
+            "2040-01-02",
```

Comment:
```suggestion
            "2023-01-02",
            "2034-01-02",
```
Let's go into the future as little as possible.